tools/github/generate-release-body.ts

```diff
@@ -1,83 +1,33 @@
 import { execSync } from "child_process";
-import { readFileSync } from "fs";
 import yargs from "yargs";
-import path from "path";
+import { getCompareLink } from "./github-utils";
 
 function capitalize(s) {
   return s[0].toUpperCase() + s.slice(1);
 }
 
-function getCompareLink(packageName: string, previousTag: string, newTag: string) {
-  const previousPackage = execSync(
-    `git show ${previousTag}:../Cargo.lock | grep ${packageName}? | head -1 | grep -o '".*"'`
-  ).toString();
-  const previosCommit = /#([0-9a-f]*)/g.exec(previousPackage)[1].slice(0, 8);
-  const previousRepo = /(https:\/\/.*)\?/g.exec(previousPackage)[1];
-
-  const newPackage = execSync(
-    `git show ${newTag}:../Cargo.lock | grep ${packageName}? | head -1 | grep -o '".*"'`
-  ).toString();
-  const newCommit = /#([0-9a-f]*)/g.exec(newPackage)[1].slice(0, 8);
-  const newRepo = /(https:\/\/.*)\?/g.exec(newPackage)[1];
-  const newRepoOrganization = /github.com\/([^\/]*)/g.exec(newRepo)[1];
-
-  const diffLink =
-    previousRepo !== newRepo
-      ? `${previousRepo}/compare/${previosCommit}...${newRepoOrganization}:${newCommit}`
-      : `${previousRepo}/compare/${previosCommit}...${newCommit}`;
-
-  return diffLink;
-}
-
-function getRuntimeInfo(srtoolReportFolder: string, runtimeName: string) {
-  const specVersion = execSync(
-    `cat ../runtime/${runtimeName}/src/lib.rs | grep 'spec_version: [0-9]*' | tail -1`
-  ).toString();
-  return {
-    name: runtimeName,
-    version: /:\s?([0-9A-z\-]*)/.exec(specVersion)[1],
-    srtool: JSON.parse(
-      readFileSync(path.join(srtoolReportFolder, `./${runtimeName}-srtool-digest.json`)).toString()
-    ),
-  };
-}
-
 const main = () => {
   const argv = yargs(process.argv.slice(2))
     .usage("Usage: npm run ts-node github/generate-release-body.ts [args]")
     .version("1.0.0")
     .options({
-      "srtool-report-folder": {
+      from: {
         type: "string",
-        describe: "folder which contains <runtime>-srtool-digest.json",
+        describe: "current tag to draft",
```

Comment:
The description isn't clear. Maybe replace with "tag of the previous release (to retrieve commit diff from)"